Nos/Views/AuthorCard.swift

```diff
@@ -71,9 +74,13 @@ struct AuthorCard: View {
                         }
                     }
                 }
+                .padding(.bottom, 12)
+
                 KnownFollowersView(author: author)
+                    .padding(.top, -8)
```

Comment:
I certainly don't like a `-8` padding here but couldn't find a better solution. Maybe one of you can. The issue I ran into was that when KnownFollowersView is an EmptyView, there wasn't enough padding below the bio. So I had to pad that enough, but then that meant unpadding this. 🤷

Reply:
Yes, it's weird. In general, I don't like to put conditional views like KnownFollowersView in a Stack because when they are empty views the still consume space (this is not the case because the spacing of the stack is zero, but I have fixed cases in which it wasn't) or force us to do gymnastics.

So, a solution here would be to move the followers request to this view and present KnownFollowersView just if the list is not empty.

But I think that would be too much maybe? You already found a valid workaround and I don't believe the advantage of doing correctly like I suggested outweighs the time it consumes doing it.

Reply:
Good point; that could be a good longer-term solution. I agree that for now it's probably not worth the time.

Reply:
Can you put `KnownFollowersView` in a `VStack` and add padding to that? Then if `KnownFollowersView` is `EmptyView()` the padding will still be rendered?

Reply:
@mplorentz I think I tried that and it didn't work, but I'll try again just to be sure.

Reply:
I just checked, and `KnownFollowersView` is currently in a `VStack`. I added `.padding(.bottom, 12)` to that `VStack` and it didn't work. Well, to clarify, it adds the padding to the bottom in both cases (empty and non-empty `KnownFollowersView`), but there's too much padding when the non-empty `KnownFollowersView` appears.

Reply:
It's not worth going back and changing it at this point, but in the future I think that be solved with `VStack(spacing: 0)`.

Reply:
I'd love to fix this, so I checked, but it looks like `KnownFollowersView` is already in a `VStack(spacing: 0)`. Were you suggesting that's the problem, or that we may need an additional `VStack`?

Reply:
Checked again and I'm not seeing another VStack here, so I'm really not sure what's going on.

Reply:
I think I just fixed it in #1183. I just added bottom padding to KnownFollowersView and it looks good to me. There is still a weird amount of space under some people's bios but that's because we are truncating the bios to 3 lines and lots of people put double newlines in their bios so the 3rd line is just whitespace.